quickannotator/db/models.py

```diff
@@ -83,11 +84,11 @@ class Tile(Base):
     tile_id = Column(Integer, nullable=False)
 
     # columns
-    pred_status = Column(Integer, nullable=True, default=TileStatus.UNSEEN)
-    pred_datetime = Column(DateTime, nullable=True, server_default=func.now())
+    pred_status = Column(Integer, nullable=False, default=TileStatus.UNSEEN)
+    pred_datetime = Column(DateTime, nullable=True)
 
-    gt_counter = Column(Integer, nullable=True, default=0)
-    gt_datetime = Column(DateTime, nullable=True, server_default=func.now())
+    gt_counter = Column(Integer, nullable=False, default=0)
```

Comment:
Did you intend for the gt_counter to be nullable? Normally I'd think the default state of a counter would be 0

Reply:
i did - my thinking was that for consistency, if gt_datetime is Null, then gt_counter should be null as well. 

having a gt_counter of "0" seems to suppose that the tile does indeed have a GT. i like, don't know what a counter =0 but a datetime = Null "means" -- it seems like a weird stage

Reply:
Okay, gt_counter is nullable again with the default value set to null.